src/FsLex.Core/fslexast.fs

```diff
@@ -92,15 +97,15 @@ let EncodeChar(c:char): Parser<uint32> = fun ctx ->
                  specificUnicodeCharsDecode.[idx] <- c
              specificUnicodeChars.[c]
      else
-         if x >= 256u then failwithf "the Unicode character '%x' may not be used unless --unicode is specified" <| int c
+         if x >= 256u then failwithf "the Unicode character '0x%x' may not be used unless --unicode is specified" <| int c
```

Comment:
Before: `FSLEX: error FSL000: the Unicode character '178' may not be used unless --unicode is specified`
After: `FSLEX: error FSL000: the Unicode character '0x178' may not be used unless --unicode is specified`

This resolves the ambiguity between https://www.compart.com/en/unicode/U+0178 and https://www.compart.com/en/unicode/U+00B2